Cargo.toml

```diff
@@ -57,6 +57,7 @@ dbpnoise = { version = "0.1.2", optional = true }
 pathfinding = { version = "4.2.1", optional = true }
 num = { version = "0.4.0", optional = true }
 dmi = { version = "0.3.0", optional = true }
+dmi2svg = { git = "https://github.com/tigercat2000/dmi2svg", optional = true }
```

Comment:
this should be [set to a release](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories) using `rev` or `tag`